src/network/__init__.py

```diff
@@ -1,3 +1,6 @@
+"""
+Network subsystem packages
```

Comment:
Why plural? `pybitmessage.network` is a package

Reply:
Yes, you're right. My bad. Thank you